src/tools/ScenarioMeasurement/Startup/PerfCollect.cs

```diff
@@ -147,7 +148,14 @@ public void AddKernelKeyword(KernelKeyword keyword)
 
         private bool LttngInstalled()
         {
-            return File.Exists("//usr/bin/lttng");
+            ProcessStartInfo procStartInfo = new ProcessStartInfo("modinfo", "lttng_probe_writeback");
+            Process proc = new Process() { StartInfo = procStartInfo, };
+            proc.StartInfo.RedirectStandardOutput = true;
+            proc.Start();
+            proc.WaitForExit();
+            string result = proc.StandardOutput.ReadToEnd();
+            // If the lttng_probe_writeback module is installed, the modinfo output will include the filename field
+            return File.Exists("//usr/bin/lttng") && result != null && result.Contains("filename:");
```

Comment:
```suggestion
            return result.Contains("filename:");
```

As we know that the existence of `//usr/bin/lttng` is poor indicator, we don't need to bother checking for it. Additionally, [`StreamReader.ReadToEnd()`](https://docs.microsoft.com/en-us/dotnet/api/system.io.streamreader.readtoend?view=net-5.0#System_IO_StreamReader_ReadToEnd) returns an empty string in the case that there was nothing there, not `null`. Thus the null check is unnecessary.